packages/web-app-files/tests/unit/components/SideBar/Shares/Links/__snapshots__/DetailsAndEdit.spec.ts.snap

```diff
@@ -71,7 +71,7 @@ exports[`DetailsAndEdit component if user can edit renders dropdown and edit but
             </oc-button-stub>
           </li>
           <li class="oc-rounded oc-menu-item-hover">
-            <oc-datepicker-stub class="link-expiry-picker" locale="en" min-date="Wed Apr 01 2020 00:00:00 GMT+0000 (Coordinated Universal Time)"></oc-datepicker-stub>
+            <!---->
```

Comment:
What happened here?

Reply:
I totally don't get what's up here - the component should be rendered or stubbed but instead it's just rendered empty - odd